ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1098,29 +1094,41 @@ class JITModBuilder {
   //
   // FIXME: this assumes that there is a single call-site of the control point.
   // https://github.com/ykjit/yk/issues/479
-  static tuple<CallInst *, Value *> GetControlPointInfo(Module *AOTMod) {
+  static tuple<CallInst *, size_t, Value *>
```

Comment:
I think we need to update the doc string to talk about the `size_t` including what it means if the `for` loop goes over bounds (i.e. doesn't call `break`).

Reply:
The for loop cannot exit by normal means by construction.

We are saying "Get the block that contains instruction I and then iterate over the block until you see I".

I experimented with debug only assertions, but it really cluttered up the code, so I removed it.

Will update the doc string though.

Reply:
> The for loop cannot exit by normal means by construction.

That's not obvious to me -- an `assert` would surely be a good thing here (or, at least, a comment).

Reply:
I'll show you what the assert looks like and you can choose :)

Reply:
Doc string: a45b81512da07e09d32b8dc791e3a3c86089c31e
Assertion: 033ab519d4f9ece83f82e44ca5c832372805be54

Reply:
I must admit, I find the assertion a little redundant. It's like saying: check that a member of a set is in that set.

Reply:
Ironically, the assert helped me realise that! So I agree that this was me being silly and that 033ab51 should be reverted.

Reply:
Reverted in e6b3b7907eff1d82ae86e6b522345d42425733b0